config/config_test.go

```diff
@@ -74,10 +74,11 @@ func TestLoadConfig(t *testing.T) {
 		t.Parallel()
 		c, err := LoadConfig("testdata/cfg/generate.yml")
 		require.NoError(t, err)
-		require.Equal(t, c.Generate.Suffix.Mutation, "Bar")
-		require.Equal(t, c.Generate.Suffix.Query, "Foo")
-		require.Equal(t, c.Generate.Prefix.Mutation, "Hoge")
-		require.Equal(t, c.Generate.Prefix.Query, "Data")
+		require.Equal(t, "Empty", c.Generate.UnamedPattern)
```

Comment:
I inverted the arguments because the testing library uses `t testing.T, expected interface{}, actual interface{}`. By inverting them, we gain a clearer message when test fails.

If you agree, I would make the flip each time I play in a test file.

Reply:
ok! I agree with your suggestion.